README.md

```diff
@@ -43,7 +43,7 @@ When you receive your Lessonly MacBook, start here before setting up other tools
         /usr/local/Cellar/salt/3003/libexec/bin/pip3.9 install m2crypto
         ```
         > Note: your salt version may be different so tab complete to replace 3003 with the version you have installed
-    - This sometimes fails to restart nginx and dnsmasq. If this happens, run the above command a second time. It usually works then.
+    - This sometimes fails to restart nginx and dnsmasq. If this happens, run the command `sudo bin/lldevconfig` again. It usually works then.
```

Comment:
If `sudo bin/lldevconfig` fails with the m2crypto error or fails to restart nginx/dnsmasq we will need to rerun the command. Thoughts on adding similar wording to line 41?
maybe 
> You will need to run the following command and then re-run `sudo bin/lldevconfig`

Reply:
Ahh I misunderstood what `This` referred to on line 46.  I thought it was the `install m2crypto` command.
I'll add that to 41.  What do you think about pulling out the 2nd bullet under #8 and making it a #9?

> Run the command again `sudo bin/lldevconfig`.  The script sometimes fails to restart nginx and dnsmasq and running it again usually fixes it.

Reply:
I think the m2crypto failure is consistent at this point so I would be okay with pulling it out into a step 9.
I would probably keep the "allow nginx popup" and "if you want to see what all ..." bullet points under step 8 though.

Reply:
I made both m2crypto and rerun the script as their own #'s.  Thoughts/opinions? Does that make sense and is it correct?